### PR TITLE
prettify helm chart before submitting to community helm chart repository

### PR DESCRIPTION
**Pull request by rimusz**
## Description
Prepare the Helm chart for submission to upstream Helm repository

## How Has This Been Tested?
Successfully run on minikube

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ X] My code follows the code style of this project.
- [ X] My change requires a change to the documentation.
- [X ] I have updated the documentation accordingly.
- [ X] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by alexellis** [comment]
@rimusz looks like we need DCO (documented in contribution guide)

File: HELM.md [hunk 1]
```diff
@@ -58,7 +49,7 @@ Clone the faas-netes repo including the helm chart:
 
 ```
 $ git clone https://github.com/openfaas/faas-netes && \
-  cd faas-netes
+  cd faas-netes/chart
```
Comment (alexellis) [inline HELM.md]: This was originally called `chart` but it broke the helm linter. This is why it's called `openfaas` instead... thoughts?

File: HELM.md [hunk 2]
```diff
@@ -111,8 +102,8 @@ By default services will be exposed with following hostnames (can be changed, se
 
 All control plane components can be cleaned up with helm with:
 
-```bash
-helm delete --purge openfaas
+```
```
Comment (alexellis) [inline HELM.md]: Why remove the `bash` indicator?

File: chart/openfaas/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+- https://github.com/openfaas/faas-netes
+home: https://www.openfaas.com
+icon: https://raw.githubusercontent.com/openfaas/media/master/OpenFaaS_logo.png
+keywords:
+- functions
+- service
+maintainers:
+- name: rimusz
+  email: rmocius@gmail.com
+- name: alexellis
```
Comment (alexellis) [inline chart/openfaas/Chart.yaml]: I think my name should be first as the author / maintainer unless you're committing to being accountable for this? 😆

File: chart/openfaas/values.yaml [hunk 4]
```diff
@@ -1,8 +1,12 @@
 functionNamespace:
 async: false
 armhf: false
+
+# Service
```
Comment (alexellis) [inline chart/openfaas/values.yaml]: Why are we deleting the comments?

File: HELM.md (hunk 1 above)
Comment (rimusz) [reply HELM.md]: strange, never had issues putting charts to `charts` folder

File: HELM.md (hunk 2 above)
Comment (rimusz) [reply HELM.md]: there should bash indicators everywhere or not at all, just to keep code consistent :-)

File: chart/openfaas/values.yaml (hunk 4 above)
Comment (rimusz) [reply chart/openfaas/values.yaml]: I did not delete comments, I added more of them

**Comment by rimusz** [comment]
close in favour of the new PR

**Pull request closed without merging**